medusa/providers/torrent/rss/nyaa.py

```diff
@@ -48,15 +48,15 @@ def search(self, search_strings, age=0, ep_obj=None, **kwargs):
 
         :param search_strings: A dict with mode (key) and the search value (value)
         :param age: Not used
-        :param ep_obj: Not used
+        :param ep_obj: An episode object
         :returns: A list of search results (structure)
         """
         results = []
 
         # Search Params
         search_params = {
             'page': 'rss',
-            'c': '1_0',  # All Anime
+            'c': '1_0' if ep_obj.series.is_anime else '4_0',  # All Anime
```

Comment:
Test is failing, because we have tests where we test an rss/daily search. Those don't pass an ep_obj. So you have to take that into account for both of your PR's.

Reply:
Just did a commit on the AniDex one. Doing this one now.

I opted to handle the None case in the provider, instead of fixing the test case to pass an object, since it seems sensible to handle all the cases in the function itself (And I failed my attempt at making the test work :p )